src/main/java/frc/robot/subsystems/drive/SwerveModuleIOTalonFXSim.java

```diff
@@ -127,29 +133,42 @@ public void config() {
         final InvertedValue driveInvertedValue = InvertedValue.CounterClockwise_Positive;
         driveTalonFXConfiguration.Slot0 = new Slot0Configs()
-                .withKV(0.973);
-        driveTalonFXConfiguration.TorqueCurrent.PeakForwardTorqueCurrent = 60;
-        driveTalonFXConfiguration.TorqueCurrent.PeakReverseTorqueCurrent = -60;
+                .withKP(50)
+                .withKS(4.796)
+                .withKA(2.549);
+        driveTalonFXConfiguration.TorqueCurrent.PeakForwardTorqueCurrent = Constants.Swerve.Modules.SLIP_CURRENT_A;
+        driveTalonFXConfiguration.TorqueCurrent.PeakReverseTorqueCurrent = -Constants.Swerve.Modules.SLIP_CURRENT_A;
+        driveTalonFXConfiguration.CurrentLimits.StatorCurrentLimit = Constants.Swerve.Modules.SLIP_CURRENT_A;
+        driveTalonFXConfiguration.CurrentLimits.StatorCurrentLimitEnable = true;
         driveTalonFXConfiguration.ClosedLoopRamps.TorqueClosedLoopRampPeriod = 0.2;
-        driveTalonFXConfiguration.Feedback.FeedbackSensorSource = FeedbackSensorSourceValue.RotorSensor;
         driveTalonFXConfiguration.Feedback.SensorToMechanismRatio = Constants.Swerve.Modules.DRIVER_GEAR_RATIO;
-        driveTalonFXConfiguration.MotorOutput.NeutralMode = NeutralModeValue.Coast;
+        driveTalonFXConfiguration.MotorOutput.NeutralMode = NeutralModeValue.Brake;
         driveTalonFXConfiguration.MotorOutput.Inverted = driveInvertedValue;
         driveMotor.getConfigurator().apply(driveTalonFXConfiguration);
 
         final InvertedValue turnInvertedValue = InvertedValue.Clockwise_Positive;
         turnTalonFXConfiguration.Slot0 = new Slot0Configs()
-                .withKP(40);
-        turnTalonFXConfiguration.Voltage.PeakForwardVoltage = 6;
-        turnTalonFXConfiguration.Voltage.PeakReverseVoltage = -6;
+                .withKP(30)
+                .withKS(0.5);
+        turnTalonFXConfiguration.TorqueCurrent.PeakForwardTorqueCurrent = 40;
+        turnTalonFXConfiguration.TorqueCurrent.PeakReverseTorqueCurrent = -40;
         turnTalonFXConfiguration.Feedback.FeedbackRemoteSensorID = turnEncoder.getDeviceID();
         turnTalonFXConfiguration.Feedback.FeedbackSensorSource = FeedbackSensorSourceValue.FusedCANcoder;
         turnTalonFXConfiguration.Feedback.RotorToSensorRatio = Constants.Swerve.Modules.TURNER_GEAR_RATIO;
-        turnTalonFXConfiguration.ClosedLoopGeneral.ContinuousWrap = true;
-        turnTalonFXConfiguration.MotorOutput.NeutralMode = NeutralModeValue.Coast;
+        turnTalonFXConfiguration.MotorOutput.NeutralMode = NeutralModeValue.Brake;
         turnTalonFXConfiguration.MotorOutput.Inverted = turnInvertedValue;
+
+        turnTalonFXConfiguration.MotionMagic.MotionMagicCruiseVelocity =
+                100.0 / Constants.Swerve.Modules.TURNER_GEAR_RATIO;
+        turnTalonFXConfiguration.MotionMagic.MotionMagicExpo_kV = 0.12 * Constants.Swerve.Modules.TURNER_GEAR_RATIO;
+        turnTalonFXConfiguration.MotionMagic.MotionMagicExpo_kA = 0.1;
```

Comment:
`MotionMagic` configurations should be removed when not using motion magic control modes

Reply:
Removed.